jsapp/js/components/drawer.es6

```diff
@@ -180,9 +179,6 @@ var Drawer = React.createClass({
         </div>
       </bem.Drawer>
       );
-  },
-  componentDidUpdate() {
-    mdl.upgradeDom();
```

Comment:
Another reason this PR needs to be merged... no more calls to `mdl.upgradeDom()` _every render_ !